src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalAmount = days * 40;
+
+  if (days < 3) {
+    return totalAmount;
+  }
+
+  if (days < 7) {
+    return totalAmount - 20;
+  } else {
+    return totalAmount - 50;
+  }
```

Comment:
Good job!
But lets make some variables for values, because for now we don't understand what it is - 40, 50, 20 ...